### PR TITLE
Add amazee.io as featured sponsor, optasy as standard sponsor

### PR DESCRIPTION
**Pull request by rfay**
## The Issue

* Add Amazee.io as featured sponsor 🚀 
* Add Optasy as standard sponsor 🎉 

Unfortunately the regular Optasy logo has apparently fixed white space around it so it ends up way too small. They sent .ai files for the logo, attached.

[optasy_api_files.zip](https://github.com/ddev/ddev.com/files/13183260/optasy_api_files.zip)

### CODE REVIEW

**Comment by bmartinez287** [comment]
Screenshot 2023-10-26 at 8 48 07 PM
Don't forget to add the  "dark logo" key.   In some cases, it can be the same as the light mode one.

**Comment by bmartinez287** [comment]
optasy
optasy-darkmode

Here are the two Optasy files. https://www.optasy.com/
I got them from the logo on their website.

**Comment by rfay** [comment]
Thanks @bmartinez287 , I'm afraid I don't know how to make a dark logo for these things. Do you know how to get me through this one? Your help would be appreciated. A followup would be fine also. But the biggest problem is the optasy

Weird that if they had the svg they gave me the .ai, oh well :)

**Comment by bmartinez287** [comment]
Yeah, I will add it here. It's fairly simple, in most cases is just making the text all white.

**Comment by rfay** [comment]
Pushed your new ones, thanks.

**Comment by bmartinez287** [comment]
[Logos-new-sponsors.zip](https://github.com/ddev/ddev.com/files/13184017/Logos-new-sponsors.zip)
Here is the zip file.

**Comment by bmartinez287** [comment]
Screenshot 2023-10-26 at 9 04 05 PM
Looks better but don't forget to add

```
"logo": "/logos/optasy_full_logo.svg",
 "squareLogo": "/logos/optasy_square.svg",
 "darklogo": "/logos/optasy_darkmode.svg",

```

**Comment by bmartinez287** [comment]
Screenshot 2023-10-26 at 9 17 46 PM
Looks good in dark mode and light mode.

**Comment by bmartinez287** [comment]
[optasy-square.svg.zip](https://github.com/ddev/ddev.com/files/13184153/optasy-square.svg.zip)
This one is better for the square, it could be better but it removes the text.

File: src/featured-sponsors.json [hunk 1]
```diff
@@ -96,6 +105,15 @@
     "url": "https://mobilistics.de/",
     "github": "mobilistics"
   },
+  {
+    "name": "optasy",
+    "type": "standard",
+    "logo": "/logos/optasy-full.svg",
+    "darklogo": "/logos/optasy-darkmode.svg",
+    "squareLogo": "/logos/optasy-square.svg",
```
Comment (bmartinez287) [inline src/featured-sponsors.json:113]: the preview shows the wrong file with this name. It's odd. Since the right one was on a previous PR.

**Comment by rfay** [comment]
Looks OK to me know, OK with you @bmartinez287 ?

**Comment by rfay** [comment]
Let me know when you're ok with this @bmartinez287

**Comment by bmartinez287** [comment]
Looks good to me.

**Review by bmartinez287 — APPROVED** [review]

**Pull request merged**